org.gnome.Music.json

```diff
@@ -163,8 +157,8 @@
             ],
             "sources": [{
                 "type": "archive",
-                "url": "https://github.com/kjd/idna/archive/v2.6.tar.gz",
-                "sha256": "53c722c4b7908dfdf2e5db2b79982f1084494db7b34fd31ff6a296e9fddfceaa"
+                "url": "https://github.com/kjd/idna/archive/v2.7.tar.gz",
+                "sha256": "2a1026d4532b1712819eec4447f8eb46639692570f31edb73e33286c888edbcc"
```

Comment:
This all got dropped: idna/requests/urllib/chardet/certifi. Got all replaced by libsoup. See upstream json @ https://gitlab.gnome.org/GNOME/gnome-music/blob/gnome-3-30/org.gnome.Music.json.

Reply:
Fixed those comments here https://github.com/flathub/org.gnome.Music/pull/5
Btw, it would be nice if you could update the application here too as the manifest is up to date on GNOME Music repository :) 
I don't follow GNOME Music's dev that closely, so I don't always have an idea of what was added/changed/removed.